src/charts.js

```diff
@@ -552,13 +565,13 @@ window.submitForm = function submitForm() {
 
         if (packageNames.length >= 1 && packageNames[0].trim() !== '') {
             formData['package'] = $.map(packageNames, function (packageName) {
-                return packageName.trim();
+                return packageName.trim().toLowerCase();
             });
         } else {
             formData['package'] = ['clone'];
         }
     } else if ($nameType.val() == 'author') {
-        var authorName = $('input[name=author]').val();
+        var authorName = $('input[name=author]').val().trim().toLowerCase();
```

Comment:
I don't know whether that's also true for author names. Trimming is fine, though.

Reply:
Do you think we can try keeping `toLowerCase` here, removing it if anyone files a bug? The fact that npm forces lowercase usernames on sign-up is a pretty big indication that this behavior will be correct most of the time, if not all of the time. If we don’t coerce lowercase here, then additional UI would need added to suggest trying lowercase when a username with mixed or upper case isn't found.

Reply:
Never mind.. i’ll make the change as requested. 

I emailed NPM Support, who replied:

> Thanks for reaching out.

> You can't assume all usernames are lowercase. Lowercase usernames were disallowed from the namespace approximately two years ago. Prior to that action, it was possible to create mixed-case usernames and some of those accounts may remain in the namespace. We do also know that mixed case usernames are pretty broken.

Sooo yeah